website/pages/en/quick-start.mdx

```diff
@@ -35,133 +29,155 @@ Using [yarn](https://yarnpkg.com/):
 yarn global add @graphprotocol/graph-cli
 ```
 
-## 3. Initialize your subgraph from existing contract
+### 2. Create Your Subgraph
+
+If your contract has events, the `init` command will automatically create a scaffold of a subgraph.
+
+#### Create via Graph CLI
+
+Use the following command to create a subgraph in Subgraph Studio using CLI:
+
+```sh
+init --product subgraph-studio
+```
+
+#### Create via Subgraph Studio
 
-Initialize your subgraph from an existing contract by running the initialize command:
+Subgraph Studio has a staging environment where you can create, manage, deploy, and publish subgraphs and API Keys.
+
+1.  Go to [Subgraph Studio](https://thegraph.com/studio/) and connect your wallet.
+2.  Click “Create a Subgraph". It is recommended to name the subgraph in Title Case: "Subgraph Name Chain Name".
+
+For additional information on subgraph creation and CLI, see [Creating a Subgraph](/developing/creating-a-subgraph).
+
+### 3. Initialize Your Subgraph
+
+#### From an existing contract
+
+The following command initializes your subgraph from an existing contract:
 
 ```sh
 graph init --studio <SUBGRAPH_SLUG>
 ```
 
-> You can find commands for your specific subgraph on the subgraph page in [Subgraph Studio](https://thegraph.com/studio/).
+> _Note: If your contract was verified on Etherscan, then the ABI will automatically be created in the CLI._
```

Comment:
```suggestion
> Note: If your contract was verified on Etherscan, then the ABI will automatically be created in the CLI.
```
The `>` should already format these "notes" in a consistent way. If we want to make that format italic, we should do it in that component.